### ISSUE TITLE
console.lua:551: attempt to concatenate local 'line' (a userdata value)

### ISSUE THREAD

**Issue by olegrok** [tarantool/tarantool#4617]
Tarantool version `2.3.0-136-gaf9ef2115`

Reproduce case:

First console:
```lua
box.cfg{listen = 3303}
uuid = require('uuid')
f = function() return function() end end
box.schema.func.create('f')
box.schema.user.passwd('admin', 'kek')
-- Go to the second console
```

Second console:
```lua
nb = require('net.box')
c = nb:connect('admin:kek@localhost:3303')
c:call('f')
c:call('f')
c:call('f')
-- several times

-- Get error message
-- Go to the first console
```

First console:
```lua
-- Press enter, some symbol and try to use autocomplete
```

**Comment by olegrok** [comment]
Looks like broken lua stack after an attempt to serialize function

**Comment by olegrok** [comment]
Reproduced on 2.3.0-203-g77fa45bd0:

```
tarantool> f2019-11-09 18:41:53.288 [7933] main utils.c:1028 E> LuajitError: builtin/box/console.lua:556: attempt to concatenate local 'line' (a userdata value)
2019-11-09 18:41:53.288 [7933] main F> fatal error, exiting the event loop
```
and Tarantool 1.10.4-29-gebd1a8200:

```
tarantool> s2019-11-09 18:45:51.232 [13862] main utils.c:966 E> LuajitError: builtin/box/console.lua:192: attempt to concatenate local 'line' (a userdata value)
2019-11-09 18:45:51.232 [13862] main F> fatal error, exiting the event loop
```
@kyukhin Please, change milestone to 1.10.5

**Comment by eljashm** [comment]
Not reproducible on Mac. What platform did you use?

**Comment by olegrok** [comment]
> Not reproducible on Mac. What platform did you use?

The main reason is UUID support. Before `uuid()` throws an error

**Comment by olegrok** [comment]
I changed my example the problem still exists.

**Comment by igormunkin** [comment]
The issue is resolved in 2.9.0-275-g027775ff2 (027775f). The fix is also backported to 2.8.1-175-g44a7ba72c (44a7ba7), 2.7.2-168-geb9344911 (eb93449), 1.10.10-70-g21b444c83 (21b444c).
